app/src/main/java/com/capstone/weatherapp/WeatherApiResponse.kt

```diff
@@ -0,0 +1,14 @@
+package com.capstone.weatherapp
+
+data class City(val coord: Coord, val weather: List<Weather>, val main: Main,
```

Comment:
let's fix the indenting here as well. let's have each parameter on its own line so its more readable, similar to what you did for the `getCities()` query parameters above